src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -79,3 +79,31 @@
     - name: Show the details for a connected kubernetes cluster
       text: az connectedk8s show -g resourceGroupName -n connectedClusterName
 """
+
+helps['connectedk8s proxy'] = """
+  type: command
+  short-summary: Get access to a connected kubernetes cluster.
+  examples:
+  - name: Get access to a connected kubernetes cluster.
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName
+"""
+
+helps['connectedk8s enable-features'] = """
+  type: command
+  short-summary: Enables the selective features on the connected cluster.
+  examples:
+  - name: Enables the Cluster-Connect feature.
+    text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features cluster-connect
+  - name: Enable multiple features.
+    text: az connectedk8s enable-features -n clusterName -g resourceGroupName --features cluster-connect custom-locations
```

Comment:
>cluster-connect custom-locations [](start = 89, length = 32)

Are the features expected to be comma separated or space separated?

Reply:
space separated, CLI offers space-separated param lists

Reply:
Its also written in enable-features --help